features/prompt-registry/function-calling.mdx

```diff
@@ -4,28 +4,31 @@ icon: "function"
 ---
 
 
-The Prompt Registry supports the [OpenAI functions](https://platform.openai.com/docs/guides/function-calling) format. You can create a prompt using functions both visually and programmatically.
+The Prompt Registry supports the [OpenAI tools](https://platform.openai.com/docs/guides/function-calling) format. You can create a prompt using tools both visually and programmatically.
 
 Be sure to read [this OpenAI guide](https://platform.openai.com/docs/guides/function-calling) for more context.
 
 ### Creating Visually
 
-Functions can be defined, called, and set up visually through the Prompt Registry.
+Tools can be defined, called, and set up visually through the Prompt Registry.
 
-<img src="/images/create-function.gif" />
+<video controls="controls">
+    <source src="/videos/create-function.mp4" type="video/mp4" />
+</video> 
 
 ### Publishing Programmatically
 
-To publish a prompt template with functions programatically is very similar to how you would publish a regular prompt template. To add functions you can add the arguments `functions` and `function_call` to your `prompt_template` object.
+To publish a prompt template with tools programatically is very similar to how you would publish a regular prompt template. To add tools you can add the arguments `tools` and `tool_choice` to your `prompt_template` object.
 
-If you’re already calling OpenAI API, you can bundle the functions and publish it into the Prompt Registry, like this:
+If you’re already calling OpenAI API, you can bundle the tools and publish it into the Prompt Registry, like this:
 
 <CodeGroup>
 
 ```python Python SDK
-messages = [{"role": "user", "content": "What's the weather like in Boston?"}]
-functions = [
-  {
+messages = [{"role": "user", "content": {"type": "text", "text": "What's the weather like in Boston?"}}]
+tools = [{
+  "type": "function",
```

Comment:
what is "type": "function"?

Reply:
@jzone3 that's how tools are defined. [See here](https://platform.openai.com/docs/api-reference/chat/create#chat-create-tools).